config/static-network-config.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "x500Name" : "CN=Alice, OU=Test Dept, O=R3, L=London, C=GB",
+    "cpi" : "cpi name"
```

Comment:
shouldn't the 'cpi' be with a more meaningful value (see the one for the notary)

Reply:
Totally. This comes in to the the needs refactoring but I don't want to change too much in one go' bucket because naming is not quite as straight forward as you would expect in this case.

Reply:
I've created a ticket for this: https://r3-cev.atlassian.net/browse/CORE-11638